hermes-consumers/src/test/java/pl/allegro/tech/hermes/consumers/supervisor/workload/MutableConfigFactory.java

```diff
@@ -0,0 +1,48 @@
+package pl.allegro.tech.hermes.consumers.supervisor.workload;
+
+import pl.allegro.tech.hermes.common.config.ConfigFactory;
+import pl.allegro.tech.hermes.common.config.Configs;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class MutableConfigFactory extends ConfigFactory {
```

Comment:
Instead of copying this class from integration module you can move it to hermes-test-helper module.